### PR TITLE
feat: Add Kleinanzeigen theme

### PR DESCRIPTION
**Pull request by kazaky**
## 📋 Changes description

Adding Kleinanzeigen theme to Spark Catalog

## 🤔 Context

Showcase Kleinanzeigen brand, prepare Polaris to support Kleinanzeigen theme, by including it in Spark design system

## ✅ Checklist

- [x] I have tested on a phone device/emulator.
- [ ] If it includes design changes, please ask for a review `spark-design` GitHub team.

## 📸 Screenshots

Screenshot 2023-08-17 at 9 18 16 AM
Screenshot 2023-08-17 at 9 18 29 AM

## 🗒️ Other info

[Contributing](https://github.com/adevinta/spark-android/blob/main/docs/contributing.md) has more information and tips for a great pull request.

### CODE REVIEW

File: catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt [hunk 1]
```diff
@@ -19,19 +19,178 @@
...
+    onBasicContainer = Green700,
+    main = Green300,
+    onMain = Green700,
+    mainContainer = Green100,
+    onMainContainer = Green700,
+    mainVariant = Green500,
+    onMainVariant = White,
+    support = Green700,
+    onSupport = White,
+    supportContainer = Green700,
```
Comment (soulcramer) [inline catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt:113]:
```suggestion
    supportContainer = Green200,
```

File: catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/*
+package com.adevinta.spark.catalog.themes.themeprovider.kleinanzeigen/*
```
Comment (soulcramer) [inline catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt:1]: The package should be bellow the copyrights

**Review by soulcramer — APPROVED** [review]

**Comment by kazaky** [comment]
Added APK for testing
[catalog-debug.apk.zip](https://github.com/adevinta/spark-android/files/12366545/catalog-debug.apk.zip)

File: catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt [hunk 3]
```diff
@@ -22,16 +22,176 @@
...
+    onMain = Green700,
+    mainContainer = Green100,
+    onMainContainer = Green700,
+    mainVariant = Green500,
+    onMainVariant = White,
+    support = Green700,
+    onSupport = White,
+    supportContainer = Green200,
+    onSupportContainer = Green700,
+    supportVariant = Green800, // tertiary => neutral / Dim 1
```
Comment (MarinaRomanova) [inline catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt:116]: probably we do not need this comment about tertiary

Comment (MarinaRomanova) [inline catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt:116]:
```suggestion
    supportVariant = Green800,
```

File: catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt [hunk 4]
```diff
@@ -22,16 +22,176 @@
...
+    onSupportVariant = White,
+    success = Green600,
+    onSuccess = White,
+    successContainer = Green100, // == surface disabled
```
Comment (MarinaRomanova) [inline catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt:120]:
```suggestion
    successContainer = Green100,
```

File: catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt [hunk 5]
```diff
@@ -22,16 +22,176 @@
...
+    onMain = Green700,
+    mainContainer = Green800,
+    onMainContainer = Green100,
+    mainVariant = Green500,
+    onMainVariant = White,
+    support = Green200,
+    onSupport = Neutral900,
+    supportContainer = Green700,
+    onSupportContainer = Green100,
+    supportVariant = Green800, // tertiary => neutral / Dim 1
```
Comment (MarinaRomanova) [inline catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt:167]:
```suggestion
    supportVariant = Green800,
```

File: catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt [hunk 6]
```diff
@@ -22,16 +22,176 @@
...
+    onSupportVariant = White,
+    success = Green100,
+    onSuccess = Neutral900,
+    successContainer = Green600, // == surface disabled
```
Comment (MarinaRomanova) [inline catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Color.kt:171]:
```suggestion
    successContainer = Green600,
```

File: catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Typography.kt [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+    subhead = subheadType,
+    body1 = body1Type,
+    body2 = body2Type,
+    caption = captionType,
+    small = smallType,
+    callout = calloutType,
+)
+
+public fun brandTypography(
+    isLegacy: Boolean,
```
Comment (MarinaRomanova) [inline catalog/src/main/kotlin/com/adevinta/spark/catalog/themes/themeprovider/kleinanzeigen/Typography.kt:213]: do we need legacy?

**Review by MarinaRomanova — APPROVED** [review]

**Pull request merged**